src/SDK/Resource/Detectors/ProcessRuntime.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\SDK\Resource\Detectors;
+
+use OpenTelemetry\SDK\Attributes;
+use OpenTelemetry\SDK\Resource\ResourceDetectorInterface;
+use OpenTelemetry\SDK\Resource\ResourceInfo;
+use OpenTelemetry\SemConv\ResourceAttributes;
+use function php_sapi_name;
+use const PHP_VERSION;
+
+/**
+ * @see https://github.com/open-telemetry/opentelemetry-specification/blob/v1.8.0/specification/resource/semantic_conventions/process.md#process-runtimes
+ */
+final class ProcessRuntime implements ResourceDetectorInterface
+{
+    public function getResource(): ResourceInfo
+    {
+        $attributes = [
+            ResourceAttributes::PROCESS_RUNTIME_NAME => php_sapi_name(),
```

Comment:
Could we also support [php-runtime/runtime](https://github.com/php-runtime/runtime)?
The term "runtime" is very ambigous in general, however in a lot of cases which could be considered a "runtime" ´php_sapi_name()` just returns `cli`.

Reply:
Is it possible to determine the runtime / runner without checking the stack trace for `RunnerInterface::run()`? (Could be unreliable depending on the usage of the runtime component.)

Should be implemented as additional detector to allow opting in/out separately.